django/db/models/query.py

```diff
@@ -732,7 +737,17 @@ def values_list(self, *fields, **kwargs):
         if flat and len(fields) > 1:
             raise TypeError("'flat' is not valid when values_list is called with more than one field.")
 
-        clone = self._values(*fields)
+        _fields = []
+        expressions = {}
+        for field in fields:
+            if hasattr(field, 'resolve_expression'):
+                field_id = str(id(field))
```

Comment:
I'm worried that in extreme edge-cases `id(field)` might clash leading to a weird failure. I'm not sure what to do instead though.